### ISSUE TITLE
Internet connection error !

### ISSUE THREAD

**Issue by 4m1n0s** [doppelgunner/YouBot#3]
i'm connected to the internet but when i try to search, i'm getting an alert from Youbot that i'm not connected to the internet

**Comment by doppelgunner** [comment]
Download the latest release for now. Also, google sent me an email and
restricted the data api. They said I violated some terms. I made an appeal
and let's wait for 2 business days and see if they aprove the app. If they
did not then this is gg lol.

**Comment by 4m1n0s** [comment]
this Application comment through Google Api ?

**Comment by doppelgunner** [comment]
What do you mean? Im using gmail to reply to your comments

**Comment by 4m1n0s** [comment]
Yes, you use Google Api to send the comments ?

**Comment by doppelgunner** [comment]
No. Github emailed me with this issue you made. So it iust popped up on
gmail. If this comment is what you are talking about. It is my repository
after all so i get notified and can post comments through gmail. I think
github made it so you can post comments through email replies.

**Comment by 4m1n0s** [comment]
No i'm talking about the fonctionality of your program, because i try to do the same thing using PHP

i followed this documantation: https://developers.google.com/youtube/v3/code_samples/php#create_and_manage_comments

did you do the same thing on your programe ?

**Comment by doppelgunner** [comment]
Im using google api yes but I used java for the programming language. So
basically it is a desktop application not a web app like yours.

**Comment by doppelgunner** [comment]
I guess if youbare trying to build one like mine maybe youll get an email
and will block your use of google api too. So if you dont care then good
luck.

**Comment by 4m1n0s** [comment]
yes, i understand you but we are both using Google Api, it mean we are using the same service with different language, the main problem is from Google Api service he will mark your application as a spam and will also hide the comments created by your application and block your use of Google Api

**Comment by doppelgunner** [comment]
Also right now i requested an appeal and they said wait for 2 business
days. Not sure if they will allow my app.

**Comment by 4m1n0s** [comment]
The best way to create a Youtube Bot it's to make it run through the Browser and to integrate a ReCAPTCHA Solver like this one on the video

https://www.youtube.com/watch?v=MeHJysMJBaY

**Comment by doppelgunner** [comment]
I used java but I think the process is the same. You need to let the user
sign in first to google then make them click allow button for obtaining the
google oauth. So yes its just the same but the languagr and syntax is
different just follow google docs for php. I just used it and try search on
google and sometimes I found solutions on stack overflow

**Comment by 4m1n0s** [comment]
Yes sir, i understand you because we are trying to do the same thing, but the problem is from Google Api Service that will always block the application :)

**Comment by doppelgunner** [comment]
Can I ask why do I need captcha resolver? I dont understand the use of that?

**Comment by doppelgunner** [comment]
Mine got blocked caused received an email that they said I violated some
rules. Before that it's working fine.

**Comment by 4m1n0s** [comment]
I will explain to you: when you send your comments through Google Api Service he will apply his spam filter on each request and if he detect that you are sending a lot of comments he will block your application and he will said "You violated some rules ",

But if you make your application run through the Browser the only method to ensure that you are not a rebot is CAPTCHA, and you need the CAPTCHA Resolver to Bypass the CAPTCH

**Comment by doppelgunner** [comment]
Ok so how do I fix these? Using a java app?

**Comment by doppelgunner** [comment]
Ok now I think my app is gg. Anyway this is just for fun.

**Comment by 4m1n0s** [comment]
Your idea is good but it doesn't work for a long time, because you will get blocked each time !

**Comment by 4m1n0s** [comment]
i don't have any experiance on Java but i will explain to you what you should do,

You need to Signin through the InAppBrowser, after that open the Youtube Video URL  and send the comment, and all of that should happen using the InAppBrowser ("A Browser inside the Application");

**Comment by doppelgunner** [comment]
Lol then I would rather make a new app than fix this one. I guess Ill try a
different app for now. Its just for fun anyway. Thanks though.

**Comment by 4m1n0s** [comment]
Yes sir i suggest to you to make a new one because with this one it will always get blocked from Google,

Thank you sir, and wich you a good luck 👍

**Comment by doppelgunner** [comment]
Is this what you mean?: 

- https://www.mkyong.com/java/how-to-automate-login-a-website-java-example/

maybe in the future I can just update the app if Im not busy and have motivation. But right now im trying to learn other stuff.

**Comment by Mehnou** [comment]
Was this fixed? @doppelgunner

**Comment by doppelgunner** [comment]
Sorry no! My appeal got rejected.
